lib/internal/modules/cjs/loader.js

```diff
@@ -188,6 +189,11 @@ for (const { 0: id, 1: mod } of NativeModule.map) {
   }
 }
 
+let allBuiltins = builtinModules.slice();
+allBuiltins = builtinModules.forEach((builtin) => allBuiltins.push(`node:${builtin}`));
```

Comment:
There shouldn't be an assignment here.

```suggestion
builtinModules.forEach((builtin) => allBuiltins.push(`node:${builtin}`));
```

Reply:
Ah!